pages/gm/f21.js

```diff
@@ -22,10 +22,26 @@ const calculateTimeStrings = ({days, hours, minutes, seconds}) => {
   return {dayString, hourString, minuteString, secondString};
 };
 
-function countdownRenderer({ days, hours, minutes, seconds }) {
+function countdownRenderer({ days, hours, minutes, seconds, completed }) {
   const {dayString, hourString, minuteString, secondString} =
     calculateTimeStrings({days, hours, minutes, seconds});
 
+  if (completed) {
```

Comment:
Is the goal with this to toggle on the page earlier than when the event happens? If not, could we do that?

Reply:
It'll toggle exactly when the timer expires, which in our case right now would at 6pm

Reply:
Think I might've misunderstood the functionality, what's deployed right now seems good to me.